docs/api.rst

```diff
@@ -3981,8 +3989,11 @@ AuditLogDiff
 
         :type: Dict[:class:`int`, :class:`ApplicationCommandPermissions`]
 
-.. this is currently missing the following keys: reason and application_id
-   I'm not sure how to about porting these
+    .. attribute:: application_id
+
+        The ID of the application that created a webhook.
+
+        :type: :class:`int`
```

Comment:
versionadded?

Reply:
`AuditLogDiff` fields generally don't have a `versionadded`, and this field in particular has always existed, just as a `str` instead of an `int`